src/CommandingOfficers/Commander.java

```diff
@@ -443,6 +443,32 @@ private void writeObject(ObjectOutputStream stream) throws IOException {
     }
   }
 
+  /** Adds 10 to commander attack upon capturing a com tower */
+  private static class TowerDamageBoostAbility extends GameEventListener
+  {
+    private Commander myCommander = null;
+
+    public LootAbility(Commander myCo)
+    {
+      myCommander = myCo;
+    }
+
+    @Override
+    public void receiveCaptureEvent(Unit unit, Location location, Commander previousOwner)
+    {
+      if( unit.CO == myCommander && location.getOwner() == myCommander)
+      {
+        // Captured a new com tower. Apply damage boost. 
+        new CODamageModifier(10).apply(this);
+      }
+      else if (unit.CO != myCommander && previousOwner == myCommander) 
+      {
+        // Friendly com tower has been captured. Revert damage boost. 
+        new CODamageModifier(10).revert(this);
```

Comment:
I know it shouldn't be possible to lose a tower if you haven't captured one first, but it will be safer to count how many towers this CO actually owns instead of trusting implicit math. For example, if each Commander starts with a Tower, that won't register here and you'll still get nerfed if you lose one.

Another approach might be: Give Commander an `adjustTowerPower` function that counts towers and creates/applies a `CODamageModifier` if needed (it would have to track the current power level explicitly, instead of implicitly as you do here). Then just call `adjustTowerPower` during `initTurn` (It would actually be sufficient to call it once at game start instead of every turn, but there's not an obvious place to do that) and also whenever `receiveCaptureEvent` is called.
This would also make it so we don't need to track the previous owner of a captured property (which shouldn't really be necessary anyway, since the new/previous owner should always be different).